Automattic/wp-calypso
# Link the Let's work on your site button to the full screen launchpad

Related to #85197

## Proposed Changes

This changes the href of the Let's work on your site button you see after you've upgraded your plan:
- If you still have tasks to be completed on the Launchpad, it links to the full screen launchpad
- Else it links to `/home`


## Testing Instructions

- Start on https://wordpress.com/start
- Start a free site, choose an intent that lands you in the build intent (other, promote, ...), choose a design and skip somewhere during the launchpad (click Skip for now)
- Upgrade your site to a paid plan (give yourself some free credits through User RC)
- You'll land on the Thank You screen. Replace `https://wordpress.com/ `in your address bar with `http://calypso.localhost:3000`

Please verify that the button links to the launchpad while there are still tasks open, and links to home when the launchpad is completed.

Beware that two tasks can't be finished in the launchpad itself:
- Write first post
- Launch your site

However, when linking to the launchpad with these tasks still open, the launchpad will redirect to /home, which is the behavior we want.


## Pre-merge Checklist
